src/contributing/style_guide.md

```diff
@@ -0,0 +1,85 @@
+# Style Guide
+
+These are some general style guidelines for articles which appear on `docs.telemetry.mozilla.org` (DTMO). Reading this will help you write material that is more useful to others.
+
+<!-- toc -->
+
+## Audience
+
+The primary audience for DTMO are *data practitioners* at Mozilla. A "data practitioner" is someone who wants to inform themselves or others using Mozilla-collected data. Here are some real-world examples of this persona:
```

Comment:
*Data practitioner* at Mozilla represent the primary audience for DTMO. They want to enable their audiences to learn about data that Mozilla has collected.